src/resources/i18n/translations/en.json

```diff
@@ -344,7 +343,7 @@
 	"Invalid or missing QR data": "Invalid or missing QR data",
 	"Invalid params in feed url": "Invalid parameters in URL",
 	"Invalid username or password": "Wrong username or password",
-	"Join our Discord": "Shitpost on our Discord",
+	"Join our Discord": "Join our Discord",
```

Comment:
```suggestion
	"Join our Discord": "We have a Discord",
```

Reply:
Alright, you probably still won't like this, but can you change it to "Chat with us on Discord"? That way we somewhat explain what Discord is and why someone would use it.